src/supermarket/app/helpers/application_helper.rb

```diff
@@ -39,4 +39,29 @@ def flash_message_class_for(name)
       "warning" => "warning",
     }.fetch(name)
   end
+
+  #
+  # Returns a github user's profile url
+  #
+  # @param name [String]
+  #
+  # @return [String]
+  #
+
+  def github_profile_url(username)
+    path = ENV["GITHUB_ENTERPRISE_URL"] || ENV["GITHUB_URL"]
+    path += "/" unless path.end_with?("/")
+    path + username
+  end
+
+  #
+  # Returns a github account title
+  #
+  # @param name [String]
+  #
+  # @return [String]
+  #
+  def github_profile_title(name)
+    "#{name} GitHub#{ENV["GITHUB_ENTERPRISE_URL"].present? ? " Enterprise" : ""} Username"
+  end
```

Comment:
We should ideally use a separate helper file for introducing specific helper methods, otherwise the application_helper will keep growing for all different use cases.

Reply:
Yes I agree, but currently there is only 5 methods including these methods, so why should we create another helper file, also these methods we are using in sidebar which is common view that's why I added it in application helper.

Reply:
ok